lbz/resource.py

```diff
@@ -111,6 +112,11 @@ def get_guest_authorization() -> dict:
         Place to configure default authorization. That will be used when Authorization Header is not in place.
         """
 
+    def get_all_possible_authz(self) -> dict:
+        return authz_collector.dump_authz(
```

Comment:
Wouldn't it be simpler to keep permissions closer to the resource? I mean something similar to `_router = Router()` that we already have 🤔